src/transactions/test/BumpSequenceTests.cpp

```diff
@@ -51,7 +51,9 @@ TEST_CASE("bump sequence", "[tx][bumpsequence]")
                 REQUIRE(a.loadSequenceNumber() == newSeq);
                 SECTION("no more tx when INT64_MAX is reached")
                 {
-                    REQUIRE_THROWS_AS(a.pay(root, 1), ex_txBAD_SEQ);
+                    REQUIRE_THROWS_AS(
+                        applyTx({a.tx({payment(root, 1)}, newSeq)}, *app),
```

Comment:
This is not right: passing the same sequence number will never work.

The test is just misleading. What we want to do here is ensure that we don't "wrap" sequence numbers (we used to use unsigned ints for sequence numbers).
So a reasonable candidate for this test is `std::numeric_limits<SequenceNumber>::min()` - assuming 2s complements etc which is the only thing we can really do.

Reply:
ah nm, passing `newSeq` is the right thing to do. It was the old behavior that was incorrect

Reply:
nope checking again, this is testing that we're doing the first part of this condition (ie that core doesn't wrap around), so we have to do what I posted above.

```
bool
TransactionFrame::isBadSeq(int64_t seqNum) const
{
    return seqNum == INT64_MAX || seqNum + 1 != getSeqNum();
}
```

Reply:
> The test is just misleading. What we want to do here is ensure that we don't "wrap" sequence numbers (we used to use unsigned ints for sequence numbers).
>So a reasonable candidate for this test is std::numeric_limits<SequenceNumber>::min() - assuming 2s complements etc which is the only thing we can really do.

It has certainly misled me!  Thanks for the suggestion.  I've made that change.